src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_archive_utils.py

```diff
@@ -220,4 +220,4 @@ def check_remote_source_code(source_location):
                 raise CLIError("'{}' doesn't exist.".format(source_location))
 
     raise CLIError(
-        "'{}' is not a valid remote URL for git or tarball.".format(source_location))
+        "[Errno 2] No such file or directory: '{0}'".format(source_location))
```

Comment:
It seems like the original logic was written only to allow remote git or tarball and had no concept of a local path. However, the help clearly calls out that local path is allowed. The new logic looks to me like it would always raise an error, even if the path DOES exist. Also, the error message should be consistent (and this looks very system specific). Line 220 uses "'{}' doesn't exist." and I would argue that is preferable.

Reply:
Make sense to me. @NatalieYuGong, can you just change the error message to `raise CLIError("'{}' doesn't exist.".format(source_location))`?